packages/twenty-front/src/pages/settings/data-model/SettingsObjects.tsx

```diff
@@ -38,13 +66,92 @@ const StyledH1Title = styled(H1Title)`
   margin-bottom: 0;
 `;
 
+const tableHeadings: TableHeading = [
```

Comment:
Something feels inconsistent here, it should be tableHeadings: TableHeading[] or tableHeading: TableHeading ; I think the singular version is best because heading is a row

Reply:
Feels like it should be TableFieldSortDefinition[] or something like that